src/Programme.java

```diff
@@ -2,6 +2,14 @@
 
     public static void main(String[] args){
         System.out.println("config");
+
+        ChuckALuck game1 = new ChuckALuck(100);
+
+        for (int i = 0; i < 10; i++) {
+            game1.playRound(1,5);
```

Comment:
Als je veel verliest, lijkt het erop dat je een negative balance kan krijgen. Is dat de bedoeling?

Reply:
Ja, dat klopt! 
De opdracht vroeg niet om dat soort dingen, dus heb ik die niet geschreven. Ik probeer mij de op het moment wat meer aan de opdracht te houden omdat ik daardoor wat meer progressie maak in de modules. Anders blijf ik te lang hangen op een onderwerp :)
Ik ben dergelijke dingen nu achteraf aan het schrijven.

Reply:
Makes sense! 👍